doc/matlab-one-line-install.md

```diff
@@ -0,0 +1,39 @@
+# One-line Installation of Robotology MATLAB/Simulink Packages
+
+This guide provides a simple documentation to install use Robotology MATLAB/Simulink packages in a pure MATLAB workflow (so without launching **anything else** via terminal, so for example no Gazebo simulation).
+
+## Installation
+~~~matlab
+websave('install_robotology_packages.m', 'https://raw.githubusercontent.com/robotology/robotology-superbuild/master/scripts/install_robotology_packages.m')
+install_robotology_packages
+robotology_setup
+~~~
+This will install everything in a `robotology-gazebo` local directory, without perturbing anything else in your system. 
```

Comment:
why does it install in a folder named `robotology-gazebo`? It is a bit confusing as in the first lines it is written that it is for a pure Matlab workflow,  `without launching **anything else** via terminal, so for example no Gazebo simulation`.

Reply:
Actually, when I tested it it created a folder named `robotology-matlab`

Reply:
Fixed in https://github.com/robotology/robotology-superbuild/pull/716/commits/f7d2d939907ea6f5ef63f74de730b876f9f68799 .